Mention other service units for the institutional network (#202)
DE-RSE/2023_paper-RSE-groups: paper.tex

The challenges of an average researchers often start a long way before reaching that level, and they might not ever need to consider HPC\@.
There are obvious reasons to closely collaborate on both consulting and training, yet at the same time a central RSE unit has to provide a much broader portfolio.

> Additionally, RSE groups might have a significant overlap with a multitude of other institutional service units.

Reviewer: ```suggestion
There are other institutional service units an RSE group needs to interact with.
```
I think overlap is not the right word since it implies that RSEs in the RSE unit do the same work. I think the idea is that the RSE unit needs to be aware of the other service units and consult their expertise where required.

> We mention the legal department for software licensing, Open Science Offices or more specifically, if existent, Open Source Policy Offices or data protection offices for work with sensitive data.

Reviewer 2: ```suggestion
We mention the legal department for software licensing, Open Science Offices or more specifically, when available, Open Source Policy Offices or data protection offices for work with sensitive data.
```
